environment-dev.yml

```diff
@@ -42,6 +41,9 @@ dependencies:
   # for release script
   - pygit2
   - pip:
+      # TODO(https://github.com/modin-project/modin/issues/5256): Move pandas back
+      # to installation with conda
+      - pandas==1.5.2
```

Comment:
This feels like _plain wrong_. This should be investigated (for example, we won't be able to make a conda-forge release doing things like that).

Reply:
I do not understand what you mean by "plain wrong". We have already been installing Ray, for instance, from PyPI in environment-dev.yml. I do not see anything bad if we install pandas from PyPI for now in order not to stop making the release in PyPI. While we are preparing the release for PyPI, we can investigate the issue in backgroung and when we come to the release in conda-forge, the issue might already be resolved. Created https://github.com/conda-incubator/setup-miniconda/issues/264. So I propose to merge the changes as is.

Reply:
I believe the fact that we mix things from `pip` and `conda` is causing us issues already - https://github.com/modin-project/modin/issues/5208 (probable cause explained around https://github.com/conda-forge/ray-packages-feedstock/pull/78#issuecomment-1316556694).

Plus it means we're not testing what our typical end user is installing, I think most customers use either `pip` or `conda` but not a semi-random mix.

Reply:
I agree that we shouldn't mix things from `pip` and `conda` but we have been doing that for a quite a long time (e.g., Ray). The fact we currenlty install pandas from `conda` and Ray from `pip` doesn't guarantee the correctness of Modin as we don't test this. Here is the similar case. My proposals are

1. Merge the changes as is.
2. Wait for a response in https://github.com/conda-incubator/setup-miniconda/issues/264 or https://github.com/pandas-dev/pandas/issues/49894.

If you have anything else, you're welcome.

I tried to use `channels` option for `conda-incubator/setup-miniconda@v2` in GH Actions to specify `conda-forge` manually and it didn't work.
I tried to remove pandas from the file and install it manually with `conda install pandas=.1.5.2 -c conda-forge` in GH Actions. It didn't work too.

Reply:
I returned back pandas to installation with conda. See more about the fix in https://github.com/conda-incubator/setup-miniconda/issues/264.